gtk4/src/subclass/widget.rs

```diff
@@ -892,48 +984,95 @@ pub unsafe trait WidgetClassSubclassExt: ClassStruct {
         }
     }
 
+    #[doc(alias = "gtk_widget_class_add_shortcut")]
     fn add_shortcut(&mut self, shortcut: &Shortcut) {
         unsafe {
             let widget_class = self as *mut _ as *mut ffi::GtkWidgetClass;
             ffi::gtk_widget_class_add_shortcut(widget_class, shortcut.to_glib_none().0);
         }
     }
 
+    #[doc(alias = "gtk_widget_class_install_property_action")]
+    fn install_property_action(&mut self, action_name: &str, property_name: &str) {
+        unsafe {
+            let widget_class = self as *mut _ as *mut ffi::GtkWidgetClass;
+            ffi::gtk_widget_class_install_property_action(
+                widget_class,
+                action_name.to_glib_none().0,
+                property_name.to_glib_none().0,
+            );
+        }
+    }
+
+    #[doc(alias = "gtk_widget_class_get_activate_signal")]
+    fn get_activate_signal(&self) -> SignalId {
+        unsafe {
+            let widget_class = self as *const _ as *mut ffi::GtkWidgetClass;
+            let signal_id = ffi::gtk_widget_class_get_activate_signal(widget_class);
+            from_glib(signal_id)
+        }
+    }
+
+    #[doc(alias = "gtk_widget_class_set_activate_signal")]
+    fn set_activate_signal(&mut self, signal_id: SignalId) {
+        unsafe {
+            let widget_class = self as *mut _ as *mut ffi::GtkWidgetClass;
+            ffi::gtk_widget_class_set_activate_signal(widget_class, signal_id.to_glib())
```

Comment:
I have no idea how this is supposed to be used :) An example for that would be good at some point, and maybe we can invent an easier API for it in Rust.

Reply:
Some widgets can be activated like buttons, check buttons, toggle buttons, switches. Using this function allows you to change the signal to be used for activation instead of the typical "activate". 

I think it's good as is for now